digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala

```diff
@@ -385,7 +385,7 @@ class VVHClient(vvhRestApiEndPoint: String) {
     * Used by RoadAddressService.getMissingRoadAddresses
     */
   def fetchByMunicipality(municipality: Int, roadNumbers: Seq[(Int, Int)]): Seq[VVHRoadlink] = {
-    val roadNumberFilters = withRoadNumbersFilter(roadNumbers, true, "")
+    val roadNumberFilters = withRoadNumbersFilter(roadNumbers, false, "")
```

Comment:
On municipality search we are searching for missing addresses -> changing this to false breaks missing address search.